commentjson/__init__.py

```diff
@@ -1,5 +1,6 @@
-from commentjson import dump
-from commentjson import dumps
-from commentjson import JSONLibraryException
-from commentjson import load
-from commentjson import loads
+__all__ = ['dump', 'dumps', 'load', 'loads', 'JSONLibraryException']
+
+try: # for python 2.x
+	from commentjson import dump, dumps, load, loads, JSONLibraryException
```

Comment:
Could be; I'm not able to test now. I just know that the original version did not work.